requirements.in

```diff
@@ -1,3 +1,8 @@
+# ======================================================================
+# If any top level dependency is added, updated or deleted be
+# sure to check pyproject.toml too. pyproject.toml is the *only*
+# dependencies file used by downstream and hence *must* be synchronized.
```

Comment:
Also, in `pyproject.toml` it's better not to pin on a specific versions, but to define a wider range.

e.g:   'django_prometheus~=2.2.0' instead of `  'django_prometheus==2.2.0',`

We don't want to force the users to have the very exact versions in their venv.

Reply:
We only have one `==` (`pydantic==2.*`) however that includes a wildcard and hence I assume does much the same.